packages/components/datagrid/src/js/datagrid.html

```diff
@@ -20,13 +20,17 @@
         <div class="oui-datagrid-responsive-container__overflow-container">
             <table class="oui-datagrid oui-datagrid_responsive">
                 <thead class="oui-datagrid__headers">
-                    <tr class="oui-datagrid__row">
+                    <tr class="oui-datagrid__row"
+                        ng-class="{'expandable': $ctrl.expandableRows}">
                         <th class="oui-datagrid__header oui-datagrid__cell-sticky"
                             ng-if="$ctrl.selectableRows">
                             <oui-checkbox model="$ctrl.selectAllRows"
                                             on-change="$ctrl.toggleAllRowsSelection(modelValue)">
                             </oui-checkbox>
                         </th>
+                        <th class="oui-datagrid__header oui-datagrid__cell-sticky expandable"
```

Comment:
Not BEM compliant:

```suggestion
                        <th class="oui-datagrid__header oui-datagrid__cell-sticky oui-datagrid__header_expandable"
```

But it seems that there is no corresponding style for `th.expandable` in your style.